authorisation.cpp

```diff
@@ -194,8 +194,13 @@ void DeRestPluginPrivate::authorise(ApiRequest &req, ApiResponse &rsp)
             {
                 // supports deCONZ specifics
             }
+            // else if (!(i->useragent.isEmpty()) && i->useragent.startsWith(QLatin1String("Mozilla"))) // Phoscon
+            // {
+            //     req.mode = ApiModeNormal;
+            // }
```

Comment:
Please remove dead code from PR :)

Reply:
Sorry... will do :)
Thanks for reviewing